### PR TITLE
Upgrading to Ruby 2.2.0

### PR DESCRIPTION
**Pull request by binarycleric**
Upgrading ruby and various gems.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 gem 'rack-jekyll' #, :git => 'https://github.com/adaoraul/rack-jekyll'
 gem 'jekyll-sass'
 gem 'jekyll-haml'
 gem 'haml-contrib'  #, :require => 'haml/filters/maruku'
+
 group :development do
-  gem "heroku"
-  gem "mr_poole" #quick way to generate posts: poole post "Title" | editor
+  gem 'rake'
+  gem 'heroku'
```
Comment (colindean) [inline Gemfile]: heroku discourages use of the gem now, right?

File: Gemfile [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 gem 'jekyll'
 gem 'rack-jekyll' #, :git => 'https://github.com/adaoraul/rack-jekyll'
 gem 'jekyll-sass'
 gem 'jekyll-haml'
 gem 'haml-contrib'  #, :require => 'haml/filters/maruku'
+
 group :development do
-  gem "heroku"
-  gem "mr_poole" #quick way to generate posts: poole post "Title" | editor
+  gem 'rake'
```
Comment (colindean) [inline Gemfile]: I don't think we even use rake at all

File: .ruby-version [hunk 3]
```diff
@@ -1 +1 @@
-ruby-2.1.2
+ruby-2.2.0
```
Comment (colindean) [inline .ruby-version]: Might as well do 2.2.1

File: Gemfile (hunk 1 above)
Comment (binarycleric) [inline Gemfile]: Seperate PR.

File: Gemfile (hunk 2 above)
Comment (binarycleric) [inline Gemfile]: Assumed we needed it because we have a `Rakefile`.

File: .ruby-version (hunk 3 above)
Comment (binarycleric) [inline .ruby-version]: Good point. Will update that shortly. Not gonna update the branch name cause I'd have to open a new PR.

**Pull request merged**